internal/integration/integration_test.go

```diff
@@ -65,17 +63,7 @@ func TestIntegration(t *testing.T) {
 }
 
 func init() {
-	var (
-		defaultTalosConfig string
-		ok                 bool
-	)
-
-	if defaultTalosConfig, ok = os.LookupEnv(constants.TalosConfigEnvVar); !ok {
-		home, err := os.UserHomeDir()
-		if err == nil {
-			defaultTalosConfig = path.Join(home, ".talos", "config")
-		}
-	}
+	defaultTalosConfig, _ := config.GetDefaultPath() //nolint: errcheck
```

Comment:
I know we have no other way other than a `panic` here, but it would be nice if we didn't have to check for an error. If we didn't have the `TALOSDIR` env var, we would have to do this I think.

Reply:
this is just building a default value, if we can't build it for whatever reason, it's not fatal, integration-test can still proceed if flag is provided explicitly, so I think panic is overkill here?